.github/scripts/setup-manager-self-hosted.py

```diff
@@ -22,12 +22,11 @@ def initialize_manager_hosted():
     # Catch any exception that occurs so that we can gracefully teardown
     try:
         # wait until machine launch is complete
-        with cd(manager_home_dir):
-            with settings(warn_only=True):
-                rc = run("timeout 20m grep -q '.*machine launch script complete.*' <(tail -f machine-launchstatus)").return_code
-                if rc != 0:
-                    run("cat machine-launchstatus.log")
-                    raise Exception("machine-launch-script.sh failed to run")
+        with settings(warn_only=True):
+            rc = run("timeout 20m grep -q '.*machine launch script complete.*' <(tail -f /machine-launchstatus)").return_code
+            if rc != 0:
+                run("cat /machine-launchstatus.log")
+                raise Exception("machine-launch-script.sh failed to run")
```

Comment:
Ok.  I understand why `expect` isn't being found for the script later.  When `machine-launch-script.sh` was installing new stuff into an existing PATH directory, after it's done, this script already has it's environment configured like it needs to be.  However, now that it is creating a new PATH directory and we really want to 'conda activate' the environment (because there are also many other variables set in the environment when you activate it), stuff lower in this script doesn't see the environment it needs to.

I'm going to look into whether it is possible to activate the environment from within python somehow (it must be).  If it isn't, then @abejgonzalez, would you be opposed to me refactoring CI a bit so that this waiter happens at the end of the step that runs machine-launch-script.sh?  If we wait to start the shell until after we know machine-launchstatus.log is complete, then everything should work as expected.

Reply:
I think we can just put a precmd on the remaining commands in the script to make it work for now.  It's ugly and probably better to refactor CI so that we follow our own advice from https://github.com/firesim/firesim/blame/use_conda/docs/Initial-Setup/Setting-up-your-Manager-Instance.rst#L82-L83

Reply:
Come to think of it, what python are you running this in?  Seems like we are possibly missing a chance to dogfood our setup by using a different python before the machine is completely setup by our script in CI...